www/src/data/sidebars/tutorial-links.yaml

```diff
@@ -120,6 +120,8 @@
           link: /tutorial/ecommerce-tutorial/
         - title: Using the WordPress Source Plugin
           link: /tutorial/wordpress-source-plugin-tutorial/
+        - title: Using the Prismic with GraphQL Source Plugin
```

Comment:
We'll very likely end up reorganizing/adding to this section, but for now I would recommend putting this after the Wordpress image tutorial to keep the two Wordpress-related things next to each other.

Reply:
Great point. Will leave Wordpress related guides together and leave it to you to restructure when necessary.